apps/src/gamelab/GameLab.js

```diff
@@ -265,7 +265,7 @@ GameLab.prototype.init = function (config) {
   config.centerEmbedded = false;
   config.wireframeShare = true;
   config.responsiveEmbedded = true;
-  config.noHowItWorks = true;
+  config.noHowItWorks = config.droplet;
```

Comment:
Nice cleanup :)
I just noticed... I don't think config.droplet is defined anywhere. Check that this still works the way you expect in gamelab. If not, take a look at "this.studioApp_.isUsingBlockly()" and see if it will do what you want.

Reply:
Chatted offline. Decided to use config.droplet as we only want to disable HowItWorks on droplet levels. Rather than disable HowItWorks for anything that isn't Blockly.